Frontend/Controllers/ProjectController.cs

```diff
@@ -62,6 +65,20 @@ private static bool HasKeyStage2PerformanceData(List<KeyStage2> keyStage2Perform
                 || HasValue(result.PercentageAchievingHigherStdInRWM)
                 || HasValue(result.PercentageMeetingExpectedStdInRWM));
         }
+        
+        private static bool HasKeyStage4PerformanceData(List<KeyStage4> keyStage2Performance)
+        {
+            return keyStage2Performance != null && keyStage2Performance.Any(result => HasValue(result.SipAttainment8score)
+                || HasValue(result.SipAttainment8scoreebacc)
+                || HasValue(result.SipAttainment8scoreenglish)
+                || HasValue(result.SipAttainment8scoremaths)
+                || HasValue(result.SipAttainment8score)
+                || HasValue(result.SipProgress8ebacc)
+                || HasValue(result.SipProgress8english)
+                || HasValue(result.SipProgress8maths)
+                || HasValue(result.SipProgress8Score)
+                || HasValue(result.SipNumberofpupilsprogress8));
+        }
```

Comment:
Do we need such a stringent check here? we needed it for KS2 data as that was _always_ returned, but is that the case with KS4 too or can we just check for items appearing in the array?